frontend/model/state.js

```diff
@@ -467,7 +483,19 @@ const actions = {
         dropAllMessagesUntilRefresh = true
       }
       if (banUser) {
-        await handleEvent.autoBanSenderOfMessage(message, e)
+        if (contractIsSyncing[contractID]) {
+          console.warn(`handleEvent: skipping autoBanSenderOfMessage since we're syncing ${contractID}`)
+        } else {
+          // NOTE: this random delay is here for multiple reasons:
+          const randomDelay = _.randomIntFromRange(0, 1500)
```

Comment:
this random could return 1, and that's not a big delay... maybe (500, 1500) is safer random number

Reply:
it's OK if there's no delay for one user as long as the delays are different between the users. It will also try again if there's a conflict